lib/ex_machina/ecto.ex

```diff
@@ -0,0 +1,58 @@
+defmodule ExMachina.Ecto do
+  defmacro __using__(opts) do
+    quote do
+      use ExMachina
+
+      @repo Dict.get(unquote(opts), :repo)
```

Comment:
Is there any way to get it to crash at compile time but also provide a friendly message? Maybe `Dict.fetch!` is good enough?